src/GitVersionCore/Core/Abstractions/Git/IReference.cs

```diff
@@ -0,0 +1,11 @@
+namespace GitVersion
+{
+    public interface IReference
```

Comment:
```suggestion
    public interface IReference : IEquatable<IReference>, IComparable<IReference>
```

Reply:
Agree, I will change that in a subsequent PR